astropy/__init__.py

```diff
@@ -65,37 +39,7 @@ def _is_astropy_source(path=None):
     online_docs_root = f'https://docs.astropy.org/en/{__version__}/'
 
 
-def _check_requirement(name, minimum_version):
-    """
-    Check that ``name`` is installed and it is of the ``minimum_version`` we
-    require.
-    """
-    # Note: We could have used distutils.version for this comparison,
-    # but it seems like overkill to import distutils at runtime, and
-    # our own utils.introspection.minversion indirectly needs requirements.
-    requirement_met = False
-    import_fail = ''
-    try:
-        module = __import__(name)
-    except ImportError:
-        import_fail = f'{name} is not installed.'
-    else:
-        version = getattr(module, '__version__')
-        requirement_met = version.split('.') >= minimum_version.split('.')
-
-    if not requirement_met:
-        msg = (f"{name} version {minimum_version} or later must "
-               f"be installed to use Astropy. {import_fail}")
-        raise ImportError(msg)
-
-    return module
-
-
-_check_requirement('numpy', __minimum_numpy_version__)
```

Comment:
Skipping these means we no longer protest if an old version of numpy or erfa is installed. I think that is probably fine...

Reply:
pip or conda will complain these days though, so think that's fine

Reply:
Agreed